media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -1988,6 +1988,63 @@ VAStatus MediaLibvaCaps::QuerySurfaceAttributes(
             i++;
         }
     }
+    else if(entrypoint == VAEntrypointEncSlice)
+    {
+        attribs[i].type = VASurfaceAttribPixelFormat;
+        attribs[i].value.type = VAGenericValueTypeInteger;
+        attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE;
+        attribs[i].value.value.i = VA_FOURCC('N', 'V', '1', '2');
```

Comment:
I realise that the only released platforms that the driver supports are old ones without 10-bit support (no KBL/CFL/GLK), but this will need to offer P010 as well on newer platforms for 10-bit encode.  (Probably fine to just ignore this for now, though.)